src/rubyHelper.ts

```diff
@@ -52,7 +52,7 @@ export class RubyHelper {
       result.options.env.RUBYLIB = '';
     }
     result.options.env.RUBY_DIR = connectionConfiguration.rubydir;
-    result.options.env.PATH =connectionConfiguration.environmentPath + result.options.env.PATH;
+    result.options.env.PATH =connectionConfiguration.environmentPath + this.pathEnvSeparator() + result.options.env.PATH;
```

Comment:
The missing path separator caused problems

Reply:
nit: should there be a space between `=connectionC`